doc/user_guide/checkers/features.rst

```diff
@@ -797,6 +797,9 @@ Refactoring checker Messages
   Emitted when a boolean condition can be simplified to a constant value.
 :simplify-boolean-expression (R1709): *Boolean expression may be simplified to %s*
   Emitted when redundant pre-python 2.5 ternary syntax is used.
+:use-yield-from (R1737): *Consider directly using 'yield from' instead*
```

Comment:
We have a distinction between "use" and "consider using", we should not mix both. Is there something to consider or is using ``yield from`` always better ?

Reply:
This check should only emit messages on very clear cases like this one:

```python
for x in gen:
    yield x
```

And in those cases I think yield from is always better. I see this message as similar to `use-dict-literal`.